### PR TITLE
Fix reported location

### PR DESCRIPTION
**Pull request by thatsmydoing**
Fixes https://github.com/jest-community/jest-runner-eslint/issues/258

### CODE REVIEW

**Comment by ljharb** [comment]
The fix is straightforward, but we'll need regression tests.

File: src/runner/__tests__/runESLint.test.js [hunk 1]
```diff
@@ -15,7 +15,20 @@ const runESLintRunnerWithMockedEngine = ({
 
       lintFiles() {
         return mockOptions.errorCount > 0
-          ? [{ errorCount: mockOptions.errorCount, warningCount: 0 }]
+          ? [
+              {
+                errorCount: mockOptions.errorCount,
+                warningCount: 0,
+                messages: [
```
Comment (thatsmydoing) [inline src/runner/__tests__/runESLint.test.js:22]: The number of entries here should probably match `mockOptions.errorCount` but for the purposes of testing, this is probably fine? The original test didn't even have `messages` to begin with.

**Review by thatsmydoing — COMMENTED** [review]
I've added a test case for it.

**Review by ljharb — APPROVED** [review]
LGTM if tests pass

**Review by SimenB — APPROVED** [review]
Thanks!

**Comment by SimenB** [comment]
I'm away from a today, but can make a release tomorrow or Monday 👍

**Comment by SimenB** [comment]
https://github.com/jest-community/jest-runner-eslint/releases/tag/v2.2.1

**Pull request merged**